.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,4 +12,5 @@
 <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
 - [ ] I have added tests covering my contributions.
-- [ ] I have updated the documentation accordingly.
\ No newline at end of file
+- [ ] I have updated the documentation accordingly.
+- [ ] This PR is being made to `staging` and not `master`
```

Comment:
Could also point out in the contribution readme where to merge with. You might be planning on doing that already..

Reply:
actually its already there under `steps to contributing` https://github.com/microsoft/ComputerVision/blob/master/CONTRIBUTING.md